crates/ra_vfs/src/io.rs

```diff
@@ -59,7 +59,7 @@ fn load_root(root: &Path, filter: &dyn Fn(&DirEntry) -> bool) -> Vec<(RelativePa
             continue;
         }
         let path = entry.path();
-        if path.extension().and_then(|os| os.to_str()) != Some("rs") {
```

Comment:
I think this if could be removed: the `filter` thing above should handle filtering.